packages/styles/scss/patterns/blocks/leadspace-block/_leadspace-block.scss

```diff
@@ -85,6 +87,8 @@
     .#{$prefix}--hr {
       border-color: $ui-04;
       margin-top: 0;
+      width: 100%;
+      max-width: carbon--mini-units(80);
```

Comment:
not sure if we should set a max-width for the horizontal rule. I'm looking at the visual specs ([https://ibm.ent.box.com/file/662154081174](https://ibm.ent.box.com/file/662154081174)) and it looks like the `hr` is longer than the `LinkList`

Reply:
Hello, Anna. Thanks for bringing this up. I'm going to fix it.

Reply:
@annawen1 as we talked on Slack, I deleted my code but the Horizontal Rule is still not touching the 12th column. This is happening because the `LeadspaceBlock` grid was changed from 12 to 8 in `lg` breakpoint in this issue: #2618.

To fix the HR width, we will be working in a separated PR.